aries-site/src/pages/templates/user-feedback-collection.mdx

```diff
@@ -0,0 +1,103 @@
+individual user should be solicited for feedback. Example engagement rules might be:
+- Has not submitted response to this questionare in the last 90 days.
+- Has not been asked to complete this questionare in the last 30 days.
+- Has not been asked to complete any questionare in the last 7 days.
+
+## Anatomy
+<Example plain>
+  <FeedbackAnatomy />
+</Example>
+
+| Label  | Region       | Purpose                                                              |                  Required                   | Notes                                                                                                      |
+| ------ | ------------ | -------------------------------------------------------------------- | :-----------------------------------------: | ---------------------------------------------------------------------------------------------------------- |
+| **1**  | **Title**    | Title prompting user for their feedback.                             | <Checkmark a11yTitle="true" size="small" /> | <span>Use the [semantically correct heading level](/foundation/typography?q=semantically#semantic-usage-of-heading-levels) for the title. "We'd love your feedback" is a nice, appreciative default title.</span>       |
+| **2**  | **Body**     | The body may be populated as many questions as appropriate.          | <Checkmark a11yTitle="true" size="small" /> | <span>Question types can include [StarRating](https://v2.grommet.io/starrating), [ThumbsRating](https://v2.grommet.io/thumbsrating), select all that apply (e.g. checkbox group), choose one (e.g. RadioButtonGroup), open ended (e.g. TextArea), or any other input types.</span> |                                                             
+| **3**  | **Footer **  | Contains two actions: "Submit" and "Dismiss" if used in Layer modal. Otherwise only "Submit" is needed inline. | <Checkmark a11yTitle="true" size="small" /> |  Submit button sends survey responses to the HPE customer experience team and initiates thank you message. If presented in a modal, the Dismiss button allows the user to bypass the survey and continue their product experience. |
+
+## Feedback setup and submission
+To iniate user feedback collection, please contact the <a href={`mailto:${contact.email}?subject=Collecting%20user%20feedback%20in%20a%20product%20experience`}>unified customer experience team</a> for the following steps:
+
+1. A dedicated survey will be created for your use case. At that time, an API token, survey id, and response id will be provided.
+1. Obtain the survey response schema with the [retrieve a survey response call](https://api.qualtrics.com/1179a68b7183c-retrieve-a-survey-response). The properties in the "values" object represent the data which should be submitted with each new survey response.
+1. Survey responses are submitted using [create a new response](https://api.qualtrics.com/f1cad92018d2b-create-a-new-response).
+
+```
```

Comment:
Remove? Or are we planning to show example "onSubmit" code here?

Reply:
Code sample added.